pkg/daemon/daemon.go

```diff
@@ -847,7 +847,8 @@ func (dn *Daemon) checkUnits(units []ignv2_2types.Unit) bool {
 
 // checkFiles validates the contents of  all the files in the
 // target config.
-func (dn *Daemon) checkFiles(files []ignv2_2types.File) bool {
+func checkFiles(files []ignv2_2types.File) bool {
+	overwrittenFiles := make(map[string]bool)
```

Comment:
Though `checkFileContentsAndMode` will still be spouting scary messages when it handles the initial instance of a path. WDYT about just reversing the list/iterating backwards and skipping over paths we've already checked?

Reply:
will do.